root/usr/bin/nm-configure

```diff
@@ -48,18 +56,19 @@ method=ignore
 [vlan]
 id=$vlanid
 EONS
-  correct_owner_perms $script
+  deploy_config $TMP_CFG $script
 elif [[ "$1" == "primary" ]]; then
   vlanid=${3:-$KCL_FDI_VLAN_PRIMARY}
   [[ "$vlanid" != "" ]] && ctype="vlan"
   script=/etc/NetworkManager/system-connections/primary
-  cat > $script <<EONS
+  cat > $TMP_CFG <<EONS
 [connection]
 id=primary
 uuid=$uuid
 type=$ctype
 autoconnect=true
 autoconnect-priority=1
+may-fail=false
```

Comment:
As far as I can tell from here, this should go in the `[ipv4]` section, shouldn't it? https://lazka.github.io/pgi-docs/NetworkManager-1.0/classes/SettingIP4Config.html